<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpScanner</class>
 <widget class="QWidget" name="IpScanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IP扫描工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,10,2">
         <item row="0" column="1">
          <widget class="QLabel" name="labelChooseRange">
           <property name="font">
            <font>
             <family>Adobe Devanagari</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择网络范围</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="ipRange">
           <item>
            <property name="text">
             <string>192.9.0.0/16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>192.9.6.0/24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>192.9.7.0/24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>192.9.10.0/24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>192.10.0.0/16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>192.168.1.0/24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>192.168.102.0/24</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="ipLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveOutput">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
